src/p5.dance.js

```diff
@@ -300,13 +300,22 @@ module.exports = class DanceParty {
       backgroundEffect.reset();
     }
 
+    let foregroundEffect = this.getForegroundEffect();
+    if (foregroundEffect && foregroundEffect.reset) {
```

Comment:
Yeah good question -- the implementation for getForegroundEffect and getBackgroundEffect are marginally different:

https://github.com/code-dot-org/dance-party/blob/2cbac648530bb387f909aae68b82b4cb887165c6/src/p5.dance.js#L336-L348

Don't quote me on this, but I _think_ this is so that we use a blank white background if there's no background, but don't fully white out the canvas if there's no foreground selected.